### ISSUE TITLE
Feature Request: RTL support

### ISSUE THREAD

**Issue by obarlev** [notaapp/nota#134]
I would be awesome if you could provide RTL support in Nota.
Currently I have no choice but to work with another Markdown editor for RTL content
Thx

**Comment by astoilkov** [comment]
I agree. This is definitely something we should do at some point. A good idea will be to implement it before we move Nota out of Beta. Thanks!
